wsgi/odyssy/odyssy/settings.py

```diff
@@ -36,6 +36,7 @@
 DEBUG = not ON_OPENSHIFT
 
 ALLOWED_HOSTS = [
+    '127.0.0.1',
```

Comment:
When it will be live we also have to turn off debug mode too. There have to be changes when the site is live. But for testing phase. It would be more generic.